scss/components/table.scss

```diff
@@ -79,6 +142,8 @@ $block: #{$fd-namespace}-table;
             border: none;
             @include fd-type("-1");
             @include fd-weight();
+            padding-top: $fd-table-header-cell-padding;
+            padding-bottom: $fd-table-header-cell-padding;
 
```

Comment:
This is the part I'm not sure about - did we want to reduce the padding here?

Reply:
Yes, this is based on Fiori 3 designs. Below is the screenshot from their master sketch file. 

<img width="606" alt="Screen Shot 2019-09-11 at 2 12 16 PM" src="https://user-images.githubusercontent.com/15215400/64727303-2f992e00-d49e-11e9-920d-e105a64a9445.png">

Reply:
Sounds good!